toolkit/tools/imageconfigvalidator/imageconfigvalidator.go

```diff
@@ -175,11 +170,6 @@ func validatePackages(config configuration.Config) (err error) {
 				return fmt.Errorf("%s: [SELinux] selected, but '%s' package is not included in the package lists", validateError, selinuxPkgName)
 			}
 		}
-		if systemConfig.EnableGrubMkconfig {
```

Comment:
I would leave this as is, don't auto install packages since that is unexpected for the user.

Reply:
We should only care if this is a non-rootfs image.